docs/content/users/quickstart.md

```diff
@@ -312,422 +268,243 @@ While the generic `php` project type is [ready to go](./project.md) with any CMS
+    ddev composer create --prefer-dist --no-scripts silverstripe/installer -y
+    ddev start
+    ddev sake dev/build flush=all
+    ```
 
-For **Drupal** and **Backdrop**, DDEV settings are written to a DDEV-managed file, `settings.ddev.php`. The `ddev config` command will ensure these settings are included in your `settings.php` through the following steps:
+=== "Git Clone"
 
-- Write DDEV settings to `settings.ddev.php`.
-- If no `settings.php` file exists, create one that includes `settings.ddev.php`.
-- If a `settings.php` file already exists, ensure that it includes `settings.ddev.php`, modifying `settings.php` to write the include if necessary.
+    ```bash
+    git clone <your-silverstripe-repo>
+    cd <your-silverstripe-project>
+    ddev config --project-type=silverstripe --docroot=public --create-docroot
+    ddev start
+    ddev composer install
+    ddev sake dev/build flush=all
+    ```
 
-For **Magento 1**, DDEV settings go into `app/etc/local.xml`.
+Your Silverstripe project is now ready.
+The CMS can be found at /admin, log into the default admin account using `admin` and `password`.
 
-In **Magento 2**, DDEV settings go into `app/etc/env.php`.
+Visit the [Silverstripe documentation](https://userhelp.silverstripe.org/en/5/) for more information.
 
-For **TYPO3**, DDEV settings are written to `AdditionalConfiguration.php`. If `AdditionalConfiguration.php` exists and is not managed by DDEV, it will not be modified.
+`ddev sake` can be used as a shorthand for the Silverstripe Make command `ddev exec vendor/bin/sake`
 
-For **WordPress**, DDEV settings are written to a DDEV-managed file, `wp-config-ddev.php`. The `ddev config` command will attempt to write settings through the following steps:
+To open the CMS directly from CLI, run `ddev launch /admin`.
 
-- Write DDEV settings to `wp-config-ddev.php`.
-- If no `wp-config.php` exists, create one that include `wp-config-ddev.php`.
-- If a DDEV-managed `wp-config.php` exists, create one that includes `wp-config.php`.
-- If a user-managed `wp-config.php` exists, instruct the user on how to modify it to include DDEV settings.
+## Statamic
 
-You’ll know DDEV is managing a settings file when you see a comment containing `#ddev-generated` like the one below. Remove the comment and DDEV will not attempt to overwrite it. If you’re letting DDEV create its settings file, we recommended leaving this comment so DDEV can continue to manage it, and make any needed changes in another settings file.
+Use a new or existing Composer project, or clone a Git repository.
 
-```
-/**
- #ddev-generated: Automatically generated Drupal settings.php file.
- ddev manages this file and may delete or overwrite the file unless this comment is removed.
- */
-```
+The Laravel project type can be used for [Statamic](https://statamic.com/) just as it can for Laravel. DDEV automatically updates or creates the `.env` file with the database information.
```

Comment:
```suggestion
The Laravel project type can be used for [Statamic](https://statamic.com/) like it can for Laravel. DDEV automatically updates or creates the `.env` file with the database information.
```